uktrade/data-hub-frontend
# Remove conditional rendering of Trade Agreement fields for Interactions

## Description of change

Previously, the Trade Agreement fields were only visible for Trade Agreement option interactions.
This field is now available regardless of the type of interaction.

There have also been minor changes with regards to casing/wording of some fields and guidance text.

## Test instructions

Attempting to add an interaction of any kind, you should now have a new mandatory field of "Related Trade Agreements".
If "Yes" is selected, you will see a multi-select field for specifying the trade agreements.
To test, enable the feature flag: `trade-agreement-interaction-v4-endpoint`

## Screenshots
### Before

![image](https://user-images.githubusercontent.com/20663545/117832216-37aa0200-b26d-11eb-8919-c05d066a14b2.png)

### After

![image](https://user-images.githubusercontent.com/20663545/117832005-03364600-b26d-11eb-91ee-feedb4218fa1.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
